### ISSUE TITLE
--ssh-config-host not working on master

### ISSUE THREAD

**Issue by Cayan** [dbcli/mycli#865]
I was excited about https://github.com/dbcli/mycli/pull/844 and decided to try it out but I got this error:

```
$ pip3 install git+https://github.com/dbcli/mycli 
$ mycli --ssh-config-host ...

Traceback (most recent call last):
  File "/home/cayan/.local/lib/python3.8/site-packages/mycli/main.py", line 1176, in cli
    ssh_config = paramiko.config.SSHConfig().from_path(ssh_config_path)
AttributeError: 'SSHConfig' object has no attribute 'from_path'
```

**Comment by pasenor** [comment]
You must have an older version of paramiko, mycli requires 2.7. 
It does not seem to be essential, I'm adding compatibility with older versions in #866, but the quick fix for now is to upgrade paramiko.

**Comment by pasenor** [comment]
fixed in version 1.22.0
